<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1162</width>
    <height>765</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="Main_Menu" native="true">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>-20</y>
      <width>1041</width>
      <height>641</height>
     </rect>
    </property>
    <widget class="QWidget" name="Saldo" native="true">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>320</y>
       <width>121</width>
       <height>71</height>
      </rect>
     </property>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>47</width>
        <height>14</height>
       </rect>
      </property>
      <property name="text">
       <string>Balance :</string>
      </property>
     </widget>
     <widget class="QLabel" name="lblAmount">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>10</y>
        <width>47</width>
        <height>14</height>
       </rect>
      </property>
      <property name="text">
       <string>Amount</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnBack2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>40</y>
        <width>80</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Back</string>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="btnLogOut">
     <property name="geometry">
      <rect>
       <x>820</x>
       <y>70</y>
       <width>80</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Log out</string>
     </property>
    </widget>
    <widget class="QWidget" name="NumMenu" native="true">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>100</y>
       <width>231</width>
       <height>211</height>
      </rect>
     </property>
     <widget class="QPushButton" name="draw20">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>90</y>
        <width>80</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>20</string>
      </property>
     </widget>
     <widget class="QPushButton" name="draw40">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>90</y>
        <width>80</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>40</string>
      </property>
     </widget>
     <widget class="QPushButton" name="draw50">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>120</y>
        <width>80</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>50</string>
      </property>
     </widget>
     <widget class="QPushButton" name="draw80">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>120</y>
        <width>80</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>80</string>
      </property>
     </widget>
     <widget class="QPushButton" name="draw100">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>150</y>
        <width>80</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>100</string>
      </property>
     </widget>
     <widget class="QPushButton" name="draw200">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>150</y>
        <width>80</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>200</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnBack">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>180</y>
        <width>80</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Back</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>60</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="Options" native="true">
     <property name="geometry">
      <rect>
       <x>119</x>
       <y>150</y>
       <width>161</width>
       <height>181</height>
      </rect>
     </property>
     <widget class="QPushButton" name="btnWithdraw">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>20</y>
        <width>80</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Withdraw</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnDeposit">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>60</y>
        <width>80</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Deposit</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnCheckBalance">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>100</y>
        <width>80</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Check balance</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnTransactions">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>140</y>
        <width>80</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Transactions</string>
      </property>
     </widget>
    </widget>
    <widget class="QListWidget" name="listTransactions">
     <property name="geometry">
      <rect>
       <x>740</x>
       <y>110</y>
       <width>256</width>
       <height>371</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1162</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
